### PR TITLE
Added some styles to make it look better

### PR DESCRIPTION
**Pull request by willianrod**
Added some styles to the site, as asked on issue #1

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -2,6 +2,7 @@ html,
 body {
   height: 100%;
   margin: 0;
+  font-family: 'Roboto', sans-serif;
```
Comment (philipp-spiess) [inline style.css]: Can we keep 'Helvetica Neue' as the main? In-fact I already set Roboto as a [fallback](https://github.com/philipp-spiess/this-week-in-react/blob/627157aae02059ae651ce728a60986afc122d600/style.css#L26) so we can just remove this line.

File: style.css [hunk 2]
```diff
@@ -104,6 +105,7 @@ h3 > a, .issues a {
   width: calc(50% - 20px);
   line-height: 60px;
   border: 3px solid white;
+  border-radius: 100px;
```
Comment (philipp-spiess) [inline style.css:107]: Nice one!

**Comment by philipp-spiess** [comment]
Thanks for your contribution, @WillianRod 🎉

File: style.css (hunk 1 above)
Comment (willianrod) [reply style.css]: Sure, no problem.

**Review by philipp-spiess — APPROVED** [review]
Thanks ❤️

**Comment by willianrod** [comment]
You're welcome

**Pull request merged**